### PR TITLE
admin: link GitHub and ORCiD icons to sites

### LINKED ISSUE #2898: admin UI fixes

### PR DESCRIPTION
**Pull request by carlinmack**
:heart: Thank you for your contribution!

Close #2898 

### Description

> Please describe briefly your pull request

Not much has changed visually besides some nicer spacing, but the icons have been linked to the user's ID on ORCiD/GitHub

I deprecated the use of `BoolFormatter` here as it would be repetitive to check whether the icon should be displayed twice

image

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [ ] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [ ] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/howtos/i18n/).
- [ ] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [ ] I've NOT included third-party code (copy/pasted source code or new dependencies).
    * If you have added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Frontend**

- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines.
- [ ] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines.
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines.

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Review by kpsherva — APPROVED** [review]

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js [hunk 1]
```diff
@@ -45,18 +45,27 @@ class SearchResultItemComponent extends Component {
...
+              <img
+                className="inline-id-icon ml-5"
+                src="/static/images/orcid.svg"
+                alt="ORCID"
+              />
+            </a>
+          )}
+          {result.identities.github && (
+            <a
+              href={`https://api.github.com/user/${result.identities.github}`}
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js:58]: Is it correct to go to the API endpoint?
```suggestion
              href={`https://github.com/${result.identities.github}`}
```

Comment (carlinmack) [reply invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js:58]: yep, going to https://api.github.com/user/6676843 works but https://github.com/6676843 does not

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js [hunk 2]
```diff
@@ -45,18 +45,27 @@ class SearchResultItemComponent extends Component {
...
+              rel="noreferrer"
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js:59]: can you fix the `rel` as commented in the other PR?
It all depends if you want to open in a new tab or not

Comment (carlinmack) [reply invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js:59]: Personally I don't think we have a [good excuse](https://css-tricks.com/use-target_blank/) to open in a new tab, I'll remove the noreferrer as we trust GitHub

**Review by anikachurilova — APPROVED** [review]

**Pull request merged**